python/ray/scripts/scripts.py

```diff
@@ -753,19 +753,28 @@ def rsync_up(cluster_config_file, source, target, cluster_name, all_nodes):
     type=int,
     help="Port to forward. Use this multiple times to forward multiple ports.")
 @click.argument("script", required=True, type=str)
-@click.option("--args", required=False, type=str, help="Script args.")
+@click.option(
+    "--args", required=False, type=str, help="(deprecated) Script args.")
```

Comment:
```suggestion
    "--args", required=False, type=str, help="(deprecated) Use '-- --arg1 --arg2' for script args.")
```

Or something like that.